react-motions/package.json

```diff
@@ -20,5 +19,9 @@
   "bugs": {
     "url": "https://github.com/raphamorim/react-motions/issues"
   },
-  "homepage": "https://github.com/raphamorim/react-motions#readme"
+  "homepage": "https://github.com/raphamorim/react-motions#readme",
+  "devDependencies": {
```

Comment:
Doesn't need to add it to the package.json.
react-motions use `yarn workspaces`, so if you're able to get all dependencies from the root. Just run `yarn` on root and all dependencies will be linked (`consider yarn@>=1.3.2`)

Reply:
this is to link the webpack with the package, since it needs a `bin` to reference the package in the main node_modules.